app/schemas/microserviceScoreCard.py

```diff
@@ -1,13 +1,18 @@
-from typing import Optional
+from typing import Optional , List
 
-from pydantic import BaseModel
+from pydantic import BaseModel 
 
 
 # Shared properties
 class MicroserviceScoreCardBase(BaseModel):
     microserviceId: Optional[int] = None
     scoreCardId: Optional[int] = None
 
+#added 
+class ScoreCardBase(BaseModel):
```

Comment:
Why is this class needed? Why don't you use the `ScoreCardInDBBase` class defined in the scoreCard.py file ?